giacomo_data/jackPapa/agents/projectron.example$.ttl

```diff
@@ -0,0 +1,16 @@
+@prefix interop: <http://www.w3.org/ns/solid/interop#>
```

Comment:
Same for projectron as with the social agents this will be in the pod of projectron. An application registration refers to this interop:application with interop:registeredAgent